frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -112,6 +117,43 @@ function FileManagerMenu:setUpdateItemTable()
     end
 
     -- tools tab
+    if Device.isKobo() then
```

Comment:
The problem is that the menu would have to be rewritten lest menu item
order is completely haphazard. That's already a bit of an issue.
On May 25, 2016 9:25 AM, "Hzj_jie" <notifications@github.com> wrote:

> In frontend/apps/filemanager/filemanagermenu.lua
> <https://github.com/koreader/koreader/pull/2045#discussion_r64526188>:
>
> > @@ -112,6 +117,43 @@ function FileManagerMenu:setUpdateItemTable()
> >      end
> >
> >      -- tools tab
> > +    if Device.isKobo() then
>
> I have a suggestion, instead of adding too many if Device.isKobo or
> similar if-else in different files, should we move these logic into device?
> For this specific case, we can add a initFileManagerMenu function in
> device, and override in kobo's device implementation.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/koreader/koreader/pull/2045/files/32d16ce93cdfa3fdd458b7c505dfca26e18034a1#r64526188>
>